skins/cat17/src/sass/layouts/pages/donation_confirmation.scss

```diff
@@ -128,9 +128,42 @@
 	}
 
 	.membership-benefits-box {
+		h2 {
+			border-bottom: solid 2px $brand-primary-membership;
+			display: inline-block;
+
+			/* Counteract styles from _layouts/memberships */
+			@include bkp(xsmax) {
+				&::after {
+					border-bottom:none;
+				}
+			}
+		}
+
 		.content {
 			padding-top: 0;
 		}
+
+		.call-to-action-box {
+			background: $brand-primary-membership;
+			color: $white;
+			border-radius: 5px;
+			padding: 15px;
+		}
+
+		.highlighted {
+			border: solid 1px $brand-primary-membership;
+		}
+
+		.list-membership li::before {
```

Comment:
Don't need to change this but quickly wanted to comment: I would personally split this into three blocks but I think we have no agreement for this and I am not sure what would be considered "best practice", I can see arguments for either way.

```
.list-membership { 
    li {
        &::before {
            color: $brand-primary-membership;
    }
}
```

Reply:
I prefer the "only nest when there are actual sub-elements" approach because it's closer to the original css notation. There is no clear rule for our frontend code.

&lt;rant&gt;
I had hoped that the original designers of the frontend would use a systematic, component/styleguide-based approach to css (like Vue components or [BEM](http://getbem.com/introduction/)). Instead, they created seemingly random css snippets, nested too deep. We did not clean up their CSS mess because we only had time for the JS mess. Instead, we continue
&/lt;rant&gt;

Let's discuss our CSS code style in the next in-person daily ...